test/e2e/backup/backup.go

```diff
@@ -58,7 +58,7 @@ func BackupRestoreTest(useVolumeSnapshots bool) {
 		var err error
 		flag.Parse()
 		UUIDgen, err = uuid.NewRandom()
-		kibishiiNamespace = "kibishii-workload" + UUIDgen.String()
+		kibishiiNamespace = "k-" + UUIDgen.String()
 		Expect(err).To(Succeed())
```

Comment:
UUIDgen is too long for the namespace, and it has some backup issues related to long strings. And I've changed it to generate random numbers in my pr